processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -7,6 +7,155 @@ import org.orbisgis.datamanager.JdbcDataSource
+                        int nbAreas = inseeCodes.size();
+                        info "$nbAreas areas will be processed"
+                        inseeCodes.eachWithIndex { code, index->
+                         if(prepareBDTopoData.execute([datasource                 : datasource,
+                                                    tableIrisName              : 'IRIS_GE', tableBuildIndifName: 'BATI_INDIFFERENCIE',
+                                                    tableBuildIndusName        : 'BATI_INDUSTRIEL', tableBuildRemarqName: 'BATI_REMARQUABLE',
+                                                    tableRoadName              : 'ROUTE', tableRailName: 'TRONCON_VOIE_FERREE',
+                                                    tableHydroName             : 'SURFACE_EAU', tableVegetName: 'ZONE_VEGETATION',
+                                                    tableImperviousSportName   : 'TERRAIN_SPORT', tableImperviousBuildSurfName: 'CONSTRUCTION_SURFACIQUE',
+                                                    tableImperviousRoadSurfName: 'SURFACE_ROUTE', tableImperviousActivSurfName: 'SURFACE_ACTIVITE',
+                                                    distBuffer                 : 500, expand: 1000, idZone: code,
+                                                    hLevMin                    : 3, hLevMax: 15, hThresholdLev2: 10
+                        ])){
+
+                             String buildingTableName = prepareBDTopoData.getResults().outputBuilding
+
+                             String roadTableName = prepareBDTopoData.getResults().outputRoad
+
+                             String railTableName = prepareBDTopoData.getResults().outputRail
+
+                             String hydrographicTableName = prepareBDTopoData.getResults().outputHydro
+
+                             String vegetationTableName = prepareBDTopoData.getResults().outputVeget
+
+                             String zoneTableName = prepareBDTopoData.getResults().outputZone
+
+                             IProcess geoIndicators = GeoIndicators()
+                             if (!geoIndicators.execute( datasource          : datasource,           zoneTable       : zoneTableName,
+                                     buildingTable       : buildingTableName,    roadTable       : roadTableName,
+                                     railTable           : railTableName,        vegetationTable : vegetationTableName,
+                                     hydrographicTable   : hydrographicTableName,indicatorUse    : indicatorUse,
+                                     svfSimplified       : svfSimplified,        prefixName      : prefixName,
+                                     mapOfWeights        : mapOfWeights)) {
+                                 error "Cannot build the geoindicators for the area $code"
+                             }
+                             else{
+                                 def tablesToSave = geoIndicators.getResults().collect {
```

Comment:
Could we add the opportunity not to save the results as in numerous IProcess (save = true by default) ? This would also be useful do add it to the OSM Workflow I think.

Reply:
I don't think so... To be discussed.

Reply:
Done the tables are saved